templates/FFV_form_factors.cpp.in

```diff
@@ -246,6 +348,96 @@ double OneLoopFunctionF(double r)
    }
 }
 
+double OneLoopFunctionH(double r)
+{
+   if (is_zero(r)) {
+      return 5.0;
+   } else if (is_zero(1.0 - r)){
+      return 0.0;
+   } else {
+      return (5.0 - 27.0 * r + 27.0 * r * r - 5.0 * r * r * r + 6.0 * (r - 3.0) * r * r * std::log(r)) /
+              pow(1.0 - r, 4);
```

Comment:
Please avoid the `pow()` function, as it is incredibly slow. Please use FlexibleSUSY's `Power4` function instead.
This also applies to the other loop functions.